### PR TITLE
work on eePlumB instructions and gather mission-date list

### PR DESCRIPTION
**Pull request by steeleb**
Some progress on the instructions! Feedback, and requests for clarity in the 'What are we labeling' section appreciated!

### CODE REVIEW

File: docs/index.Rmd [hunk 1]
```diff
@@ -6,95 +6,140 @@ output:
...
-<center>
-![](www/images/sharefolder.png)
-</center> &nbsp;&nbsp;
+   3. Create two subdirectories within this new folder - one called 'val-test' and another called 'labels'.
+
+<center>![](www/images/subfolders.png){width="400px"}</center>
+
+  
+
+3.  Once the folder is created, please share it with the group. To do this, you can click on the folder and click 'Whare.' Next, set Genera Access to 'Anyone with the link' and 'Editor'. Lastly, click 'copy link,' and paste the link in this [Google Sheets document](https://docs.google.com/spreadsheets/d/1Sk7BuA92Mw4A3_ETialeQ9b1WPXK1TWkfDd8wsvVOFc/edit?usp=sharing) alongside your name and initials.
```
Comment (lindsayplatt) [inline docs/index.Rmd:38]:
```suggestion
4.  Once the folder is created, please share it with the group. To do this, you can click on the folder and click 'Share.' Next, set General Access to 'Anyone with the link' and 'Editor'. Lastly, click 'copy link,' and paste the link in this [Google Sheets document](https://docs.google.com/spreadsheets/d/1Sk7BuA92Mw4A3_ETialeQ9b1WPXK1TWkfDd8wsvVOFc/edit?usp=sharing) alongside your name and initials.
```

File: docs/index.Rmd [hunk 2]
```diff
@@ -6,95 +6,140 @@ output:
...
+
+`darkNearShoreSediment`: dark brown or red-brown areas, usually near a stream inflow.
+
+<center>![](www/images/dNSS.png){width="400"}</center>
+
+`offShoreSediment`: could also be considered dispersed sediment. Green-ish colored areas proximate to near shore sediment. It may often look 'swirled' in the deeper areas of the lake.
+
+<center>![](www/images/oSS.png){width="400"}</center>
+
+`algalBloom`: this class is tricky! Technically, it is very hard to discriminate between off shore sediment or dispersed sediment and algal blooms in the imagery provided. In fact, it's so tricky, we don't even have an example for you! Please do use this label if you think you see a bloom.
```
Comment (lindsayplatt) [inline docs/index.Rmd:66]: > In fact, it's so tricky, we don't even have an example for you!

Perfect!

File: docs/index.Rmd [hunk 3]
```diff
@@ -6,95 +6,140 @@ output:
...
+1.  Go to the project [Github page.](https://github.com/rossyndicate/Superior-Plume-Bloom/tree/main/eePlumB/1_user_interface_script)
+
+2.  Copy the appropriate `.js` script.
+
+    -   If this is your first time through the tutorial, you will be working with the validation script: [`eePlumB_validation.js`](https://github.com/rossyndicate/Superior-Plume-Bloom/blob/5e9d0cd1ad435aa3bdb2f0d2b41c026c0f57c8ab/eePlumB/1_user_interface_script/eePlumB_validation). The purpose of this step is to have you go through the full process of classifying a few images for practice and to provide us with data to see how similarly all our volunteers classify a range of conditions. We hope that the examples below will help guide everyone to classify conditions similarly, but we understand that there will be variation as there is a level of subjectivity in this process.
+    -   If you have already completed the tutorial once using the validation script, you have made it to the main classification workflow and will be working with the classification script: [`eePlumB_classification.js`](LINK%20ME)
```
Comment (lindsayplatt) [inline docs/index.Rmd:99]: Are we at the point where we can link to the appropriate file and replace where I just wrote `LINK ME`??

**Review by lindsayplatt — COMMENTED** [review]
I didn't get a chance to render the full thing and see it because I don't have the right computer with me tonight. I followed along with the new images and changes to the .Rmd, which look good! I just added a few tiny suggestions/comments. Otherwise, I was able to follow the new section very well!

File: docs/index.Rmd [hunk 4]
```diff
@@ -6,95 +6,140 @@ output:
...
+
+`shorelineContamination`: usually dark pixels or yellow-brown pixels that overlap with the shoreline. The purpose of this label is for us to be able to add 'uncertainty' to some machine-inferred labels if they are also proximate to pixels labeled as `shorelineContamination`. Generally speaking, label things that might be 'confusing' for a machine to label because the color is similar to colors of a sediment plume. The easiest way to detect this is to turn on the 'Satellite' baselayer in the top right and toggle the 'Layer 1' option under 'Layers':
+
+<center>![](www/images/sat_toggle.png){width="400"}</center>
+
+<center>![](www/images/contamination.png){width="400"}</center>
+
+<center>![](www/images/contamination_under.png){width="400"}</center>
+
+`other`: anything else that is present in the image that might be classified as 'other' or 'unknown'.
```
[the remark this answers is not in the document]
Comment (steeleb) [reply docs/index.Rmd:88]: yes yes! I'll find an image with a red boat. :)

File: docs/index.Rmd (hunk 3 above)
Comment (steeleb) [reply docs/index.Rmd:99]: Absolutely we are, I didn't make it past the 'What are we labeling'. :) I'll be working on the remainder today!

Comment (steeleb) [reply docs/index.Rmd]: Updated in recent commits!

**Comment by steeleb** [comment]
Okay, lots here!

This PR is big progress on the following issues: #10, #14, #17, #13, #19, #20. At least some of these can be added to close with the approval and merge of this branch.

#10: 
I think this is done. Between your work and the edits, I feel like this is in a good spot. 

#14:
Also, I think we're good here. I'm sure there will be tweaks after the undergrads use it.

#17:
Again, we've got a baseline. I added some 'tips' to the end to help folks who come across empty or no-data tiles.

#13:
Again, I think we've covered a lot of this, but this is probably another one I'll get feedback on Monday about.

#19:
This is in the drive and linked in the instructions! This PR definitely closes #19.

#20:
This seems to be a duplicate of 19. I'm going to make myself feel good and close #20, too.

Please read through the instructions and let me know if they look good enough for undergrads! If you think we can close any of the other issues, please say so. :)

**Pull request merged**